### PR TITLE
Remove ntopng

### PR DESCRIPTION
**Pull request by jslobodzian**

### CODE REVIEW

File: SPECS/ntopng/ntopng.signatures.json [hunk 1]
```diff
@@ -1,6 +0,0 @@
-{
- "Signatures": {
```
Comment (PawelWMS) [inline SPECS/ntopng/ntopng.signatures.json:2]: Jon, you might also want to rever the changes in `cgmanifest.json` and the licence map for this package. #608 added `ntopng`, so I used that as a reference of what needs to be reverted.

**Comment by MateuszMalisz** [comment]
Please remember to remove the sources from the server.

File: SPECS/ntopng/ntopng.signatures.json (hunk 1 above)
Comment (jslobodzian) [reply SPECS/ntopng/ntopng.signatures.json:2]: yeah...I realized the same right after I pushed.  How does it look now?

**Review by PawelWMS — APPROVED** [review]

File: SPECS/ntopng/ntopng.signatures.json (hunk 1 above)
Comment (PawelWMS) [reply SPECS/ntopng/ntopng.signatures.json:2]: Looks good to me!

**Pull request merged**